### ISSUE TITLE
MangaDemon - same problem as always

### ISSUE THREAD

**Issue by carloshpb** [tachiyomiorg/tachiyomi-extensions#17742]
### Source information

MangaDemon 1.4.4

### Source language

English 

### Steps to reproduce

1 - Try to open any manga/comic/manhwa from MangaDemon.org at Tachiyomi;
2 - Not working at all. All updates are failing.

### Expected behavior

Get manga chapters

### Actual behavior

Not getting any chapter and getting HTTP Error 404 - probably they've changed the domain again

### Tachiyomi version

0.14.6

### Android version

Android 9

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by AwkwardPeak7** [comment]
Migrate from MangaDemon to MangaDemon to update urls.

**Comment by carloshpb** [comment]
What do you mean to migrate from one to another?

**Comment by carloshpb** [comment]
Sry I'm not really good with web scrapping. I've tried to look on the website with the Developer Tools (on Chrome), but without success in identifying what is blocking the mangas/manhwas. Those guys on MangaDemon are sniping others and filling their websites with a lot of ads (it looks like cancer to me). I've tried to identify anything different that marks the <img> tag which contains the READ AT WEBSITE image, but no success either (in mobile mode or normal mode on Chrome). Both domains are working normally in Chrome.
